clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -664,6 +664,16 @@ static void GenerateArg(SmallVectorImpl<const char *> &Args,
                     Opt.getKind(), 0, Value);
 }
 
+static void GenerateMultiArg(SmallVectorImpl<const char *> &Args,
+                             llvm::opt::OptSpecifier OptSpecifier,
+                             ArrayRef<StringRef> Values,
+                             CompilerInvocation::StringAllocator SA) {
+  Option Opt = getDriverOptTable().getOption(OptSpecifier);
+  Args.push_back(SA(Opt.getPrefix() + Opt.getName()));
```

Comment:
The single-arg case seems to call `denormalizeString`; I'm wondering if that should happen here too? Maybe @jansvoboda11 could review this bit

Reply:
> it's missing an assertion check, I'll add that.

Done.